en/docs/observability/observability-overview.md

```diff
@@ -0,0 +1,71 @@
+# Observability Overview
+
+Observability in Choreo provides the capability to visualize and monitor the performance of services deployed on Choreo. Choreo has in-built support to view the overall status, latencies, throughput data,  diagnostic data, and logs. Hence, you can use Choreo Observability to detect and troubleshoot anomalies in your services efficiently. 
+
+## Dashboard
+Choreo observability dashboard gives you more than one way to monitor your services. This section will introduce the dashboards. 
+
+### Dashboard Overview
```

Comment:
Need to follow sentence-style capitalization for all subsection headings.

Reply:
changed.